src/lib.rs

```diff
@@ -182,3 +182,21 @@ fn clockpoly(
     let arcs = [arc_outer, arc_inner].concat();
     Polygon::new(LineString::from(arcs), vec![])
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn internal() {
+        let args: Vec<String> = Vec::new();
+        let da = Params::from_iter(args);
+        // println!("{}", da); Would be good to print and test args
```

Comment:
If you're trying to do this, I think you want `{:?}` to show the debug representation

Reply:
That is a very useful trick. Still cannot see the output appearing in the output of `cargo test` but it now works, thanks!

Reply:
The output from tests is hidden unless the test fails